v2/src/modules/address/ModuleEthTxsHistory.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+    <v-row class="mx-5 my-3">
```

Comment:
There is unnecessary  left/right margins, it shift the options content to the right.